docs/help/contributor/style-guide/style-and-formatting.md

```diff
@@ -51,14 +53,36 @@ between the number and the unit.
 |4`&nbsp`CPUs          | 4 CPUs
 |14%                   | 14`&nbsp`%
 
-## Use bold for user interface elements
+### Use bold for user interface elements
 
 |:white_check_mark: Correct                  |:no_entry: Incorrect
 |--------------------|-----
 |Click **Fork** | Click "Fork"
 |Select **Other**      | Select "Other"
 
-## Use the international standard for punctuation inside quotes
+## General style
+
+### Use upper camel case for Knative API objects
+
+|:white_check_mark: Correct                  |:no_entry: Incorrect
+|--------------------|-----
+|Channels | channels
+|Broker      | broker
+|ContainerSource | Container Source
+
+### Only use parentheses for acronym explanations
+
+>Put an acronym inside parentheses after its explanation. Don’t use parentheses for anything else.
+
+>Parenthetical statements especially should be avoided because readers skip them.
+If something is important enough to be in the sentence, it should be fully part of that sentence.
+
+|:white_check_mark: Correct                  |:no_entry: Incorrect
+|--------------------|-----
+|Custom Resource Definition (CRD) |Check your CLI (you should see it there)
+|Knative Serving will create a Revision |Knative will create a Revision (a stateless, snapshot in time of your code and configuration)|
```

Comment:
```suggestion
|Knative Serving creates a Revision |Knative creates a Revision (a stateless, snapshot in time of your code and configuration)|
```
Nitpicking, but I'd prefer present tense:

https://kubernetes.io/docs/contribute/style/_print/#use-present-tense)
https://developers.google.com/style/tense?hl=en

Maybe we should add something about tenses if it's not in the guide already?

Reply:
Oh, nvm it's here 😄 https://github.com/knative/docs/pull/4035/files#diff-0138e4edfa32eb45b8ff7c58d77a845112e9bf6b6694af396b4cf53b44fc8bc4L3